COLLABORATOR_GUIDE.md

```diff
@@ -80,12 +102,33 @@ elevate discussion to the TSC for resolution (see below).
 
 Breaking changes (that is, pull requests that require an increase in
 the major version number, known as `semver-major` changes) must be
-elevated for review by the TSC. This does not necessarily mean that the
-PR must be put onto the TSC meeting agenda. If multiple TSC members
-approve (`LGTM`) the PR and no Collaborators oppose the PR, it can be
-landed. Where there is disagreement among TSC members or objections
-from one or more Collaborators, `semver-major` pull requests should be
-put on the TSC meeting agenda.
+[elevated for review by the TSC](#involving-the-tsc).
+This does not necessarily mean that the PR must be put onto the TSC meeting
+agenda. If multiple TSC members approve (`LGTM`) the PR and no Collaborators
+oppose the PR, it can be landed. Where there is disagreement among TSC members
+or objections from one or more Collaborators, `semver-major` pull requests
+should be put on the TSC meeting agenda.
+
+### Waiting for Approvals
+
+Before landing pull requests, sufficient time should be left for input
+from other Collaborators. In general, leave at least 48 hours during the
+week and 72 hours over weekends to account for international time
+differences and work schedules. However, certain types of pull requests
+can be fast-tracked and may be landed after a shorter delay:
+
+* Trivial changes, for example, those fixing minor bugs or improving
+  performance without affecting API or causing other wide-reaching impact.
```

Comment:
I'm 👎 on considering improving performance as a fast-track issue. In my experience, it's the contrary.

Reply:
This was added in https://github.com/nodejs/node/pull/16135, should we remove that one? cc @addaleax

Reply:
Also, personally I agree with @mcollina , I don't recall seeing a performance optimization PR being fast-tracked before?

Reply:
This wording was added in c52e43d97c1ee, a long, long time before my PR.

I’m okay with removing performance optimizations this as well.

Reply:
Maybe a topic for more general discussion, but at least for me the triviality of the change isn't usually a reason to fast-track. My experience is that no matter how trivial the change, there's always the possibility that someone disagrees or has nits.

For me the reason to fast-track is because it is actually beneficial, e.g.
- It's a fix we want to land in a release line
- It's a code-and-learn PR, and we want to give people a good first experience (and maybe even land on the day)
- It unbreaks CI on master

So for example if it fixes a regression but won't go into a release for a couple of weeks, what's the reason to fast-track? Same for small changes.

Reply:
@gibfahn I agree but these are just some examples, that's why we also need a rule about "n approvals to both the PR and the fast-tracking request". If a PR should not be fast-tracked, people can object to that fast-tracking request without objecting to the PR.